lib/devise_saml_authenticatable/saml_response.rb

```diff
@@ -0,0 +1,16 @@
+require 'devise_saml_authenticatable/saml_mapped_attributes'
+
+module SamlAuthenticatable
+  class SamlResponse
```

Comment:
From my own use case I have found it useful as it stores the raw SAML response in addition to the SamlMappedAttributes object. For more advanced use cases I have found I needed to delve deeper into the raw response to get the data I wanted.